packages/terra-search-field/src/SearchField.jsx

```diff
@@ -183,36 +225,57 @@ class SearchField extends React.Component {
       inputAttributes,
       ...customProps
     } = this.props;
+
+    const cancelButtonVisible = !defaultValue && !this.state.searchText ? 'input-clear' : undefined;
+
+    const cancelButtonClassNames = cx([
+      'clear',
+    ]);
+
+    const inputClassNames = cx([
+      'input',
+      cancelButtonVisible,
+    ]);
+
     const searchFieldClassNames = cx([
       'searchfield',
       { block: isBlock },
       customProps.className,
     ]);
 
-    const valueTemp = {};
-    if (value !== undefined) {
-      valueTemp.value = value;
-    } else {
-      valueTemp.defaultValue = defaultValue;
-    }
-
     const inputText = this.context.intl.formatMessage({ id: 'Terra.searchField.search' });
     const buttonText = this.context.intl.formatMessage({ id: 'Terra.searchField.submit-search' });
-    const additionalInputAttributes = Object.assign({ 'aria-label': inputText }, inputAttributes, valueTemp);
+    const additionalInputAttributes = Object.assign({ 'aria-label': inputText }, inputAttributes);
 
     return (
       <div {...customProps} className={searchFieldClassNames}>
         <Input
-          className={cx('input')}
+          defaultValue={defaultValue}
+          className={inputClassNames}
           type="search"
           placeholder={placeholder}
           onChange={this.handleTextChange}
+          onFocus={this.handleInputFocus}
           disabled={isDisabled}
           aria-disabled={isDisabled}
           onKeyDown={this.handleKeyDown}
           refCallback={inputRefCallback}
+          value={defaultValue ? undefined : this.state.searchText}
           {...additionalInputAttributes}
         />
+        { this.previouslyFocusedInput && this.state.searchText
+          ? (
+            <Button
+              className={cancelButtonClassNames}
+              onClick={this.handleClear}
+              text="cancel"
```

Comment:
We'll want to get translations for this text. I can work with you to get those from our translation team.

Reply:
I forgot to change the text when I changed other stuff from 'cancel' to 'clear'. I went back and forth on this, but clear struck me as more indicative of the intended functionality. As @bjankord said, however, if anyone has any thoughts on this, let me know!